deploy/kubernetes/driver/kubernetes/manifests/controller-server.yaml

```diff
@@ -114,6 +114,11 @@ spec:
                 configMapKeyRef:
                   name: ibm-cloud-provider-data
                   key: vpc_id
+            - name: VPC_SUBNET_IDS
+              valueFrom:
+                configMapKeyRef:
```

Comment:
We need to check if config map does not exists will deployment failed ? We should fail we need this list and config-map.

Reply:
Today we reading this via ENV variable so any updates are not considered. @skorati is working on this